### PR TITLE
[MI-740] update guzzle to 6.2.1

### PR DESCRIPTION
**Pull request by joseconsador**
/cc @zendesk/mintegrations

### Description
Addresses https://httpoxy.org/ by updating guzzle to the 6.2.1 which contains the patch.

### References
* JIRA: https://zendesk.atlassian.net/browse/MI-740

### Risks
* [HIGH] Could still be vulnerable to https://httpoxy.org/

### CODE REVIEW

**Comment by jmramos02** [comment]
👍

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-2.1.0
+2.1.1
```
Comment (miogalang) [inline VERSION]: Separate commit please

Comment (joseconsador) [inline VERSION]: @miogalang 💇

**Comment by miogalang** [comment]
👍

**Pull request merged**